extensions/geode-modules-test/src/main/java/org/apache/geode/modules/session/EmbeddedTomcat.java

```diff
@@ -39,55 +36,34 @@
 import org.apache.geode.modules.session.catalina.JvmRouteBinderValve;
 
 public class EmbeddedTomcat {
-
-  private String contextPath = null;
-  private Embedded container = null;
   private Log logger = LogFactory.getLog(getClass());
+  private int port;
+  private Embedded container;
+  private Context rootContext;
 
-  /**
-   * The directory to create the Tomcat server configuration under.
-   */
-  private String catalinaHome = "tomcat";
-
-  /**
-   * The port to run the Tomcat server on.
-   */
-  private int port = 8089;
-
-  /**
-   * The classes directory for the web application being run.
-   */
-  private String classesDir = "target/classes";
-
-  private Context rootContext = null;
-
-  private Engine engine;
-
-  /**
-   * The web resources directory for the web application being run.
-   */
-  private String webappDir = "";
-
-  public EmbeddedTomcat(String contextPath, int port, String jvmRoute)
-      throws MalformedURLException {
-    this.contextPath = contextPath;
+  EmbeddedTomcat(String contextPath, int port, String jvmRoute) throws MalformedURLException {
     this.port = port;
 
     // create server
     container = new Embedded();
-    container.setCatalinaHome(catalinaHome);
 
+    // The directory to create the Tomcat server configuration under.
+    String catalinaHome = "tomcat";
+    container.setCatalinaHome(catalinaHome);
     container.setRealm(new MemoryRealm());
 
     // create webapp loader
     WebappLoader loader = new WebappLoader(this.getClass().getClassLoader());
-    if (classesDir != null) {
-      loader.addRepository(new File(classesDir).toURI().toURL().toString());
-    }
+    // The classes directory for the web application being run.
+    String classesDir = "target/classes";
```

Comment:
Can't easily tell from this diff, but maybe you can just pass the string literal to the method call if it is the only use.

Reply:
Good catch, thanks.